tests/rptest/services/redpanda.py

```diff
@@ -497,12 +516,16 @@ def write_conf_file(self, node, override_cfg_params):
         self.logger.debug(conf)
         node.account.create_file(RedpandaService.CONFIG_FILE, conf)
 
-    def restart_nodes(self, nodes, override_cfg_params=None):
+    def restart_nodes(self,
+                      nodes,
+                      override_cfg_params=None,
+                      start_timeout=None,
+                      stop_timeout=None):
         nodes = [nodes] if isinstance(nodes, ClusterNode) else nodes
         for node in nodes:
-            self.stop_node(node)
+            self.stop_node(node, timeout=stop_timeout)
```

Comment:
to simplify, you might consider taking a single timeout and passing the remainder of the timeout to start_node

Reply:
I kind of like separating them, because when we expect a slow startup (due to lots of replay), I don't want to give slow shutdowns a free pass.

Reply:
Oh that makes sense. I didn't think about it that way!